### ISSUE TITLE
Implement Age 1 cards for Artifacts expansion

### ISSUE THREAD

**Issue by micahstairs** [micahstairs/bga-innovation#87]
This issue will track the implementation of all Age 1 cards for the Artifacts expansion. Please add a comment to this issue before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [x] **110 - Treaty of Kadesh**
  - [x] **111 - Sibidu Needle**
  - [x] **112 - Basur Hoyuk Tokens**
  - [x] **113 - Holmegaard Bows**
  - [x] **114 - Papyrus of Ani**
  - [x] **115 - Pavlovian Tusk**
  - [x] **116 - Priest-King**
  - [x] **117 - Electrum Stater of Efesos**
  - [x] **118 - Jiskairumoko Necklace**
  - [x] **119 - Dancing Girl**
  - [x] **120 - Lurgan Canoe**
  - [x] **121 - Xianrendong Shards**
  - [x] **122 - Mask of Warka**
  - [x] **123 - Ark of the Covenant**
  - [x] **124 - Tale of the Shipwrecked Sailor**

**Comment by micahstairs** [comment]
There's not currently any work up for grabs here. I'm about to file an issue to track age 2 cards.

**Comment by micahstairs** [comment]
I'll work at adding the special choice to choose the card type ("Base" or "Artifacts").
